crates/config/Cargo.toml

```diff
@@ -16,6 +16,7 @@ libp2p-identity = { workspace = true, features = ["peerid", "serde"] }
 multiaddr.workspace = true
 serde = { workspace = true, features = ["derive"] }
 toml.workspace = true
+tokio = { workspace = true, features = ["full"] }
```

Comment:
~nit: order

Reply:
and also, we don't want the `"full"` feature flag, focus on what specifically you need, in this case `"fs"`